dockstore-common/src/main/java/io/dockstore/common/MemoryIntensiveTest.java

```diff
@@ -0,0 +1,4 @@
+package io.dockstore.common;
+
+public interface MemoryIntensiveTest {
```

Comment:
Should probably have a comment on where this is used and how

Reply:
Added comment.